src/software/simulated_tests/robot_enters_region.py

```diff
@@ -1,10 +1,5 @@
-import pytest
-
-import software.geom.geometry as tbots_geom
-from proto.geometry_pb2 import Angle, AngularVelocity, Point, Vector
-from proto.validation_pb2 import ValidationGeometry, ValidationProto, ValidationStatus
-from proto.vision_pb2 import BallState, RobotState
-from proto.world_pb2 import SimulatorTick, World, WorldState
+import software.python_bindings as tbots
```

Comment:
it might be a good time to make two of these files like Arun commented on the last PR, one for friendly enters region and the other for enemy enters region

Reply:
we don't have two full systems implemented so it's unclear to me that's the architecture we'd go for